src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -1157,6 +1157,8 @@ Node.oneConnectionPerIP=Limit to one connection per address? (friends) (NOT RECO
 Node.oneConnectionPerIPLong=Automatically drop friend connections when other friends have the same (external) IP address.
 Node.opennetEnabled=Enable insecure mode (automatically connect to untrusted nodes)?
 Node.opennetEnabledLong=Enable insecure mode (aka opennet)? If this is enabled, your Freenet node will automatically exchange node references with other untrusted nodes (Strangers as opposed to Friends). But this means that the fact that you are running a node is no longer private, and many attacks are much easier. If you know enough people running Freenet, you should stick to trusted (Friends) connections to them, and turn this off.
+Node.opennetExcludedCountriesShort=Ignore Opennet peers from these countries
+Node.opennetExcludedCountriesLong=A comma separated list of country codes that will prevent your node from connecting to Opennet nodes originating in these countries.
```

Comment:
There is a giant zoo of country code standards available.
This should at least mention which standard the codes are chosen from. Ideally a link to the list of codes should be included.
Notice that ```src/freenet/l10n/ISO639_3.java``` contains an API for obtaining a list of ISO639-3 codes in case this is based on ISO639-3.